workspaces/Mynewt Nimble Host/Mynewt Nimble Host.pqw6

```diff
@@ -2610,29 +2640,41 @@
             <Mandatory>FALSE</Mandatory>
           </Row>
           <Row>
-            <Name>TSPC_SUM_31_17</Name>
+            <Name>TSPC_SUM_ICS_31_17</Name>
             <Description>Core Spec Version 4.2, Adopted 02 December 2014 (C.1)</Description>
             <Value>FALSE</Value>
             <Mandatory>FALSE</Mandatory>
           </Row>
           <Row>
-            <Name>TSPC_SUM_31_18</Name>
+            <Name>TSPC_SUM_ICS_31_18</Name>
             <Description>Core Spec Version 4.2 + HS, Adopted 02 December 2014 (C.1)</Description>
             <Value>FALSE</Value>
             <Mandatory>FALSE</Mandatory>
           </Row>
           <Row>
-            <Name>TSPC_SUM_31_19</Name>
-            <Description>	Core Specification Version 5.0. Adopted 06 December 2016 (C.1)</Description>
+            <Name>TSPC_SUM_ICS_31_19</Name>
+            <Description>Core Specification Version 5.0. Adopted 06 December 2016 (C.1)</Description>
             <Value>FALSE</Value>
             <Mandatory>FALSE</Mandatory>
           </Row>
           <Row>
-            <Name>TSPC_SUM_31_20</Name>
+            <Name>TSPC_SUM_ICS_31_20</Name>
             <Description>Core Specification Version 5.1. Adopted 15 January 2019 (C.1)</Description>
             <Value>FALSE</Value>
             <Mandatory>FALSE</Mandatory>
           </Row>
+          <Row>
+            <Name>TSPC_SUM_ICS_31_21</Name>
+            <Description>Core Specification Version 5.2. Adopted 31 December 2019 (C.1)</Description>
+            <Value>FALSE</Value>
+            <Mandatory>FALSE</Mandatory>
+          </Row>
+          <Row>
+            <Name>TSPC_SUM_ICS_31_22</Name>
+            <Description>Core Specification Version 5.3. Adopted 06 July 2021 (C.1)</Description>

```

Comment:
This probably should be updated through Launch Studio, to verify consistency. We should support the latest spec.

Reply:
It was updated via launch studio, but this had to be bumped manually. I also enabled Periodic Adv. BTW, I think this is a bug in PTS that doesn't change this ICS based on ICS file at workspace generation, because if you look here: https://github.com/intel/auto-pts/pull/719/commits/f0852c516de6e2c9f63943f49e698a418b3b0ca4#diff-0d9cc24875a692b2cbb450fa778bdf36f59ed35df9f4c0f22b0c0d171cecfc68L714 you can clearly see that it was changed from 31_21 to 31_22.